package com.aditya.dataconnect.executionengine.exceptionhandler;

public class DataConnectClientException extends Exception {


    public DataConnectClientException(String message, Throwable cause) {
        super(message, cause);
    }
}
